src/main/webapp/app/overview/exercise-chatbot/exercise-chatbot.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnDestroy } from '@angular/core';
+import { faCommentDots } from '@fortawesome/free-solid-svg-icons';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { ChatbotPopupComponent } from './chatbot-popup/chatbot-popup.component';
+import { ExerciseChatWidgetComponent } from 'app/overview/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component';
+import { Overlay } from '@angular/cdk/overlay';
+
+@Component({
+    selector: 'jhi-exercise-chatbot',
+    templateUrl: './exercise-chatbot.component.html',
+    styleUrls: ['./exercise-chatbot.component.scss'],
+})
+export class ExerciseChatbotComponent implements OnDestroy {
+    public chatAccepted = 'false';
```

Comment:
Why is this a string and not a boolean?

Reply:
Must be a misunderstanding, changed

Reply:
Sorry, this was done to get the value of the button after it is clicked, which is a string. But changed to be controlled in a different way and made boolean instead